src/util.cpp

```diff
@@ -77,11 +77,7 @@
 #endif
 
 #include <boost/algorithm/string/case_conv.hpp> // for to_lower()
-#include <boost/algorithm/string/join.hpp>
-#include <boost/algorithm/string/predicate.hpp> // for startswith() and endswith()
```

Comment:
starts_with() is [still used](https://github.com/practicalswift/bitcoin/blob/d399e1b1cf60577b2f74e1f72393ea132b4c80f4/src/util.cpp#L401).